source/extensions/filters/http/ext_proc/mutation_utils.h

```diff
@@ -14,8 +14,8 @@ namespace ExternalProcessing {
 class MutationUtils : public Logger::Loggable<Logger::Id::filter> {
 public:
   // Convert a header map until a protobuf
```

Comment:
into?

Reply:
If we change the other thing here, then I'll certainly do it -- if not, then I'd like to do it later.

Reply:
i believe it was a suggestion on fixing a typo
```suggestion
  // Convert a header map into a protobuf
```